tests/test_pycodestyle.py

```diff
@@ -39,9 +39,9 @@ def test_style(self):
                                     max_line_length=MAX_LINE_LENGTH,
                                     exclude=EXCLUDE)
         report = sg.check_files(dirs_and_files)
-        self.assertEqual(report.total_errors, ALLOWED_ERRORS,
-                         msg='{0} style violations were found. Expected {1}'.format(report.total_errors,
-                                                                                    ALLOWED_ERRORS))
+        self.assertTrue(report.total_errors <= ALLOWED_ERRORS,
```

Comment:
Please don't make this change. Please change the value of `ALLOWED_ERRORS` instead. The goal is to reduce the number of style errors in the code. By changing this from (effectively) `==` to `<=` you're giving flexibility to future authors to add style errors after you have reduced the number of style errors.

Perhaps you could put a comment in this code that tells future programmers _not_ to change the comparison, but instead to change the value of ALLOWED_ERRORS. There is already a comment with ALLOWED_ERRORS indicating that it should never be increased, only decreased.

Reply:
Made the requested fix. See 89b070d5c12048375977205f754c84bafc04bbe7